tools/ci_build/build.py

```diff
@@ -1774,6 +1780,8 @@ def run_ios_tests(args, source_dir, config, cwd):
                 dynamic_framework_dir,
                 "--framework_info_file",
                 framework_info_file,
+                "--variant",
+                "Mobile",
```

Comment:
note: test_apple_packages.py is run within `run_ios_tests` function which is only enabled when `args.ios` +`args.test` is set. so skip checking if `args.ios` is set here.